Easydict/NewApp/Configuration/Configuration+Defaults.swift

```diff
@@ -52,6 +52,7 @@ extension Defaults.Keys {
 
     static let appearanceType = Key<AppearenceType>("EZConfiguration_kApperanceKey", default: .followSystem)
     static let fontSizeOptionIndex = Key<UInt>("EZConfiguration_kTranslationControllerFontKey", default: 0)
+    static let selectedMenuBarIcon = Key<UInt>("EZConfiguration_kSelectedMenuBarIconKey", default: 0)
```

Comment:
We can just use enum `MenuBarIconType` here instead of its raw value which gives a more clear presentation.

Reply:
We should set a different default value for menubar icon in debug and release mode.

Reply:
We should set a different default value for MenuBarIcon in debug and release mode.